src/Search.php

```diff
@@ -288,7 +291,7 @@ public function hasIndexByName(string $index): bool
     public function getQuery(): Query
     {
         if (null === $this->_query) {
-            $this->_query = Query::create('');
+            $this->_query = new Query(new MatchAll());
```

Comment:
I assume this should not change anything for users?

Reply:
Yes, that's the same thing - I just made it explicit.